src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -74,10 +75,10 @@ public class CommandTestUtil {
     static {
         DESC_AMY = new EditPersonDescriptorBuilder().withName(VALID_NAME_AMY)
                 .withPhone(VALID_PHONE_AMY).withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY)
-                .withGender(VALID_GENDER_AMY).build();
+                .withDob(VALID_DOB_AMY).withGender(VALID_GENDER_AMY).build();
```

Comment:
I think the dob field can be moved after gender for parameter order consistency
```suggestion
                .withGender(VALID_GENDER_AMY).withDob(VALID_DOB_AMY).build();
```

Reply:
thx for catching!